### PR TITLE
feat: replace tailwind's config units to EM

### PR DESCRIPTION
**Pull request by huygn**

### CODE REVIEW

File: packages/utils/src/styles/tailwind-unit-replace.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        val = replacer(value);
+        break;
+    }
+    replacedTheme[key] = val;
+  }
+
+  return replacedTheme;
+}
+
+export const replaceTailwindUnit = (opts: Options) => (userConfig: TailwindConfig): TailwindConfig => {
```
Comment (zlatanpham) [inline packages/utils/src/styles/tailwind-unit-replace.ts:36]: Do you think adding a unit test for this function would be useful to be better understanding how it works?

**Review by zlatanpham — COMMENTED** [review]
LGTM. As long as you can fix the failing tests, it's all good to merge.

**Review by zlatanpham — DISMISSED** [review]
LGTM 👍 . Can you also add a changeset file by running `yarn changeset` on your local before we merge this?

**Review by zlatanpham — APPROVED** [review]

**Comment by zlatanpham** [comment]
@wwalser the code looks good to me. Please take a look if you have any suggestions, otherwise, I think we can ship it.

**Comment by wwalser** [comment]
My hope for this change is that we would not introduce significant mutations at runtime. 
That said, I'm keen enough to get this shipped that we should move forward as-is.

**Review by wwalser — APPROVED** [review]

**Comment by huygn** [comment]
> My hope for this change is that we would not introduce significant mutations at runtime.
> That said, I'm keen enough to get this shipped that we should move forward as-is.

This one runs in build time, `twin.macro` (and `tailwindcss`) reads it for configuration, the content of this file won't be included in the final build

**Comment by wwalser** [comment]
Oh, I misunderstood how this was included into adjacent packages. Nice!
Thanks for clarifying.

**Pull request merged**